src/Inputs/scene.inputManager.ts

```diff
@@ -65,7 +66,7 @@ export class InputManager {
     private _alreadyAttachedTo: HTMLElement;
 
     // Pointers
-    private _wheelEventName = "";
+    private _wheelEventName = "wheel";

```

Comment:
why do we need to keep this?

Reply:
I originally kept this just to serve as a constant string rather than use "wheel" in several places

Reply:
but do we use it in that file

Reply:
I just changed the refs to this to use the string literal and are no longer storing this value since we aren't transforming this.